tests/test_job.py

```diff
@@ -14,23 +14,17 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-from __future__ import absolute_import
-from __future__ import print_function
-
 from io import BytesIO
 from time import sleep
 
 import io
 
 from tests import testlib
 
-try:
-    import unittest2 as unittest
-except ImportError:
-    import unittest
+import unittest
 
-import splunklib.client as client
-import splunklib.results as results
+from splunklib import client
+from splunklib import results
 
 from splunklib.binding import _log_duration, HTTPError
 
```

Comment:
There's code on line 33 below which refers to Python 2.6 - recommend determining whether the comment is misleading and can now be removed, or whether more work is needed to catch ParseErrors as well.

Reply:
we had a look and it seems safe to just remove the comments.